cni/network/multitenancy.go

```diff
@@ -276,7 +276,7 @@ func getInfraVnetIP(
 	infraSubnet string,
 	nwCfg *cni.NetworkConfig,
 	plugin *NetPlugin,
-) (*cniTypesCurr.Result, error) {
+) (cniResults []cniTypesCurr.Result, err error) {
```

Comment:
getInfraVnetIP is unused func. No need to make any changes to it.

Reply:
let's remove this func

Reply:
Yes Ashvin, I remember TM suggested to leave this func here.

Reply:
I also think we should remove it, because in the check below, we are checking `if ipv4result` ... `else { do ipv6 stuff }` but won't the result be both Ipv4 and Ipv6?